### DISCUSSION TITLE
Cultural and technological exchange between the Wikimedia and open-source ecosystems

### DISCUSSION THREAD

**Discussion by epicfaace** [PlaintextGroup/oss-virtual-incubator#39] [Proposal Discussion]
Discussion for https://github.com/PlaintextGroup/oss-virtual-incubator/blob/main/proposals/Wikimedia-and-the-open-source-ecosystem.md

**Comment by mlinksva** [comment]
Copying over my [comment](https://github.com/PlaintextGroup/oss-virtual-incubator/pull/37#issuecomment-1126537045) which is probably more appropriate here.

I'm enthusiastic about this proposal. The "why innovative" section correctly points out a persistently barely tapped opportunity that the proposal would help with:

> This project matches the strengths of the Wikimedia platform to the OSS sector, and brings resources of the OSS sector into the Wikimedia platform. Although Wikipedia was established in 2001, and despite deep ideological values alignment in the Open Movement, the software sector has few partnership precedents with Wikimedia. There should be more cultural exchange between OSS and Wikimedia, and this proposal is innovative for presenting actionable ways to collaborate along these lines.

Further, my sense is that more/better info abut OSS in Wikimedia projects (particularly Wikidata) would lower the barrier to better global characterizations of OSS, in part by providing richer, accessible, consistent, cross-referenceable facts about notable OSS projects and their developers/organizations that otherwise aren't available or require lots of expensive curation that is not shared across research questions, and therefore is not gathered in the first place.

**Comment by Daniel-Mietchen** [comment]
@epicfaace @mlinksva We developed one of the 13 proposals &mdash; the Wiki99 one &mdash; a bit further, which resulted in
- a [video](https://www.youtube.com/watch?v=11_ATt0Vm38) introducing the Wiki99 concept
- a first draft of a [list](https://meta.wikimedia.org/wiki/Wiki99/software) of software-related topics and their coverage across a selection of Wikipedia languages.

The number of entries is currently not 99, nor is the selection of languages or section headings finished, so as to underline that there is room for discussion and refinement taking into account further opinions. Of course, this could be scaled up or down as needed.

**Reply by epicfaace** [reply]
Thanks for sharing! Taking a look.

**Reply by epicfaace** [reply]
@Daniel-Mietchen this is very interesting, thanks for sharing! I'm curious about, in particular, what are the kinds of articles do you think that US policymakers (e.g. Congressional staffers interested in learning what OSS is) would be looking at? For example, they might not be likely to look at https://en.wikipedia.org/wiki/Compiler but would very likely look at https://en.wikipedia.org/wiki/Log4Shell. Additionally, to focus in on _quality_ of English-level articles, would it be useful, do you think, to list out which articles are B-class, C-class, etc.?

**Comment by epicfaace** [comment]
Very interesting article demonstrating the importance of wikipedia -- https://news.mit.edu/2022/study-finds-wikipedia-influences-judicial-behavior-0727 -- "Getting a public Wikipedia article increased a case’s citations by more than 20 percent. The increase was statistically significant"
